app/controllers/users.py

```diff
@@ -123,15 +123,34 @@ def get(self):
         page = request.args.get('page', 1,type=int)
         per_page = request.args.get('per_page', 10, type=int)
         keywords = request.args.get('keywords' , '')
+        verified = request.args.get('verified' , True)
+        is_beta = request.args.get('is_beta' , False)
+        All = request.args.get('All' , False)
 
         users = []
 
         if (keywords == ''):
-            paginated = User.find_all(paginate=True, page=page, per_page=per_page)
-            users = paginated.items
-            pagination = paginated.pagination
+            if not All :
+                paginated = User.query.filter(User.verified == verified , User.is_beta_user == is_beta).order_by(User.date_created.desc()).paginate(page=page, per_page=per_page, error_out=False)
```

Comment:
This part of the query will filter by those who are both verified and beta_user variables, what if one just wants to vary just one of the variables independently? E.g if I just want to know unverified users regardless of whether or not they are beta. So I would think there would be need to separate the filter queries for each scenario, what do you think?

Reply:
Oooh ok i had not looked at that angle when i was implementing the solution . Let me cater for it also . Thanks for that @MutegekiHenry